odoo/http.py

```diff
@@ -1322,9 +1322,6 @@ def load_addons(self):
         self.dispatch = DisableCacheMiddleware(app)
 
     def setup_session(self, httprequest):
-        # recover or create session
-        session_gc(self.session_store)
```

Comment:
don't forget to remove `def session_gc()`

Reply:
I guess it can go away as it doesn't make much sense anymore if it keeps the random check.